pkg/network/setup/netstat_test.go

```diff
@@ -103,14 +103,35 @@ var _ = Describe("netstat", func() {
 			setup.NetStat.UpdateStatus(setup.Vmi, setup.Domain)
 
 			Expect(setup.Vmi.Status.Interfaces).To(Equal([]v1.VirtualMachineInstanceNetworkInterface{
-				newVMIStatusIface(primaryNetworkName, []string{primaryPodIPv4, primaryPodIPv6}, "", "", netvmispec.InfoSourceDomain),
-				newVMIStatusIface(secondaryNetworkName, []string{secondaryPodIPv4, secondaryPodIPv6}, "", "", netvmispec.InfoSourceDomain),
+				newVMIStatusIface(primaryNetworkName, []string{primaryPodIPv4, primaryPodIPv6}, "", "", netvmispec.InfoSourceDomain, netsetup.DefaultInterfaceQueueCount),
+				newVMIStatusIface(secondaryNetworkName, []string{secondaryPodIPv4, secondaryPodIPv6}, "", "", netvmispec.InfoSourceDomain, netsetup.DefaultInterfaceQueueCount),
 			}), "the pod IP/s should be reported in the status")
 
 			Expect(setup.NetStat.PodInterfaceVolatileDataIsCached(setup.Vmi, primaryNetworkName)).To(BeTrue())
 			Expect(setup.NetStat.PodInterfaceVolatileDataIsCached(setup.Vmi, secondaryNetworkName)).To(BeTrue())
 		})
 
+		It("run status and expect interface/network to be reported with the right queue count (without guest-agent)", func() {
+			var queueCount uint = 8
```

Comment:
Should this be `int32`?

Reply:
no because it must be ref here as *uint 
`domainSpecInterface.Driver = &api.InterfaceDriver{Name: "vhost", Queues: &queueCount}``

Reply:
Ahh, yes, this is an older API value. :(